addons/web/static/tests/helpers/mock_server.js

```diff
@@ -777,7 +777,7 @@ export class MockServer {
                 } else if (aggregateFunction === "week") {
                     return `W${date.toFormat("WW yyyy")}`;
```

Comment:
for symetry's sake, to parse a week number we should use ISO week, that is: "WW kkkk"
@brboi